### DISCUSSION TITLE
VitePress blog

### DISCUSSION THREAD

**Discussion by fractalf** [vuejs/vitepress#3701] [Show and tell]
Hi,
firstly, thanks a lot to the VitePress team for providing this! :)

Finally a "blog framework" that suits me. 

* markdown
* static site generator
* vue to customize

What more can you ask for?

So I set up a new blog today here
https://abc.fractalf.net

The first article is a fairly long tutorial about the process I went through to get it setup the way I wanted as a blog.
https://abc.fractalf.net/articles/001-vitepress-as-a-blog/

I've included some tweaks/customizations that I haven't seen documented anywhere else so it should be worth the read

![image](https://github.com/vuejs/vitepress/assets/5707457/cebce624-9031-420b-a679-dc5416aec25c)

Cheers!
-Alf

**Comment by rzwns** [comment]
man, how do i get this? which repo to use?

**Reply by fractalf** [reply]
Hi! I haven't cleaned up my code yet to be ready for release, but I will. Soon ..

But, most of what I have done can be easily reproduced if you follow the steps in my first article 
https://abc.fractalf.net/articles/001-vitepress-as-a-blog/

If you need any help, just ask here in discussion :)

**Reply by maks-rafalko** [reply]
@fractalf have you already open sourced it?

**Comment by intersecato** [comment]
@fractalf your guide seems to be incomplete, VPArticle.vue is missing and there is a bit of confusion...
